karma-test-shim.js

```diff
@@ -0,0 +1,64 @@
+// Tun on full stack traces in errors to help debugging
```

Comment:
The thing is. Angular 2 is really picky with how tests run. You basically need to tell karma to stop. Run some "browser adapters" on angular 2 and then run the tests. That is how Angular does it on their repo.

https://github.com/juliemr/ng2-test-seed/blob/master/karma-test-shim.js

I modified it a bit with our needs. But basically what angular does in:

https://github.com/angular/angular/blob/master/test-main.js